src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/add-polling-source-form.types.ts

```diff
@@ -0,0 +1,29 @@
+export interface JsonFormValidators {
+    required?: boolean;
```

Comment:
I guess this does not have to be undefined. This is a more or less universal constraint - being required or optional. Just having a boolean value should be enough.